app/models/intake.rb

```diff
@@ -345,14 +347,6 @@ def get_or_create_spouse_auth_token
     new_token
   end
 
-  def get_or_create_requested_docs_token
-    return requested_docs_token if requested_docs_token.present?
-
-    new_token = SecureRandom.urlsafe_base64(8)
-    update(requested_docs_token: new_token, requested_docs_token_created_at: Time.now)
-    new_token
-  end
-
```

Comment:
I was super confused by this deletion at first. It looks like this was a duplicate method, correct?

Reply:
Yes, the rebase duplicated it for some reason